apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.ts

```diff
@@ -37,8 +37,7 @@ router.get(
   '/',
   createAuthzMiddleware({
     oneOfPermissions: ['has_course_instance_permission_view'],
-    errorMessage: 'Access denied (must be a student data viewer)',
-    unauthorizedUsers: 'block',
+    unauthorizedUsers: 'passthrough',
```

Comment:
This might've been discussed in https://github.com/PrairieLearn/PrairieLearn/pull/12496, but for my understanding, why was this updated?

Reply:
This was a genuine error in refactoring!

```suggestion
    unauthorizedUsers: 'block',
```